guides/preprocessing_layers.py

```diff
@@ -294,7 +291,7 @@
 seen during `adapt()`). You can configure this by using the `mask_token` and `oov_token`
 constructor arguments  of `StringLookup`.
 
-You can see the `StringLookup` and `CategoryEncoding` layers in action in the example
+You can see the `StringLookup` in action in the example
 [structured data classification from scratch](https://keras.io/examples/structured_data/structured_data_classification_from_scratch/).
```

Comment:
Suggestion:

```suggestion
You can see the `StringLookup` in action in the
[Structured data classification from scratch](https://keras.io/examples/structured_data/structured_data_classification_from_scratch/)
example.
```

(Using sentence case)